doc/source/PreConfiguredSites.rst

```diff
@@ -388,6 +379,8 @@ As of spring 2023, there is an inconsistency in `libstdc++` versions on Acorn be
 NOAA Parallel Works (AWS, Azure, Gcloud)
 ----------------------------------------
 
+**Note. This section needs updating from EPIC**
```

Comment:
this can be removed after my other two comments are resolved (right?)

Reply:
Done!